pages/index.vue

```diff
@@ -2,16 +2,24 @@
   <div>
     <!-- Sign in/out buttons -->
     <VBtn v-if="!authStore.isLoggedIn" color="success" @click="signIn">
-      Logg inn
+      {{ $t('sign_in') }}
+    </VBtn>
+
+    <VBtn v-else color="error" @click="signOut">
+      {{ $t('sign_out') }}
     </VBtn>
-    <VBtn v-else color="error" @click="signOut">Sign out</VBtn>
 
     <p>User from store: {{ user }}</p>
     <p>User from api: {{ data }} {{ error }}</p>
 
     <VBtn color="info" @click="createUser">Create user</VBtn>
     <VBtn color="error" @click="deleteUser">Delete user</VBtn>
     <VBtn color="success" @click="refresh">Refresh user data</VBtn>
+
+    <select v-model="locale" style="cursor: pointer">
```

Comment:
Har et egen issue på å gjøre dette om til sin egen komponent, så slipper vi å hardkode inn over alt.

Reply:
Yes, it should be removed. It's just there to make it easier for anyone checking this out to test that it's working.